doc/nrf/device_guides/working_with_nrf/nrf70/developing/sniffer_rx_operation.rst

```diff
@@ -0,0 +1,84 @@
+
+.. _ug_nrf70_developing_monitor_mode_operation:
+
+Monitor mode operation
+**********************
+
+The nRF70 Series device supports IEEE 802.11 packet reception when Monitor mode is enabled in the nRF Wi-Fi driver.
+To enable the Monitor mode of operation, you can use the ``NET_REQUEST_WIFI_MODE`` network management runtime API.
+This API can be used to disable Monitor mode in the driver when IEEE 802.11 packet reception is not required by the sniffer application.
+
+If the nRF70 device is configured to operate in Monitor mode, Station mode connection is not permitted and the nRF70 Series device operates in a pure sniffer mode.
+
+When Monitor mode is enabled, you need to configure the operating channel.
+This channel will be used as the default channel on which the nRF70 device captures 802.11 packets.
+When the device operates in Monitor mode, all 802.11 packets received on the configured channel will be sent up the stack by the nRF70 Series device and the Wi-Fi driver.
+
+To set the desired channel for 802.11 packet reception, you can use the ``NET_REQUEST_WIFI_CHANNEL`` network management API.
+
+See the :ref:`wifi_shell_sample` sample for more information on configuring mode and channel settings for raw packet reception through shell commands.
+
+The following table lists the shell commands and network management APIs used to switch between operational modes and set channels:
+
+.. list-table:: Wi-Fi packet reception network management APIs
+   :header-rows: 1
+
+   * - Network management APIs
+     - Wi-Fi shell command
+     - Example usage
+     - Description
+   * - net_mgmt(NET_REQUEST_WIFI_MODE)
+     - ``wifi mode -i<interface instance> <configuration>``
+     - ``wifi mode -i1 -m``
+     - Configure interface instance 1 to Monitor mode
+   * - net_mgmt(NET_REQUEST_WIFI_MODE)
+     - ``wifi mode -i<interface instance> <configuration>``
+     - ``wifi mode -i1 -mt``
+     - Configure interface instance 1 to Monitor mode and TX injection mode
+   * - net_mgmt(NET_REQUEST_WIFI_CHANNEL)
+     - ``wifi channel -i<interface instance> <channel number to set>``
+     - ``wifi channel -i1 -c6``
+     - Set the raw transmission channel to ``6`` for interface 1
+
+.. _ug_nrf70_developing_monitor_mode_receive_packet_metadata:
+
+Monitor mode receive packet metadata
+************************************
+
+<TODO> - need to determine if radiotap header format is to be used or proprietary header.
+
+.. _ug_nrf70_developing_monitor_mode_receive_operation:
+
+Monitor mode receive operation
+******************************
+
+The IEEE 802.11 packet captured by the nRF70 Series device will be sent up to the Wi-Fi driver which will incorporate an additional radio information header to the received packet and send it to the upper layer application.
+The packet can be received in the upper layer application by opening a raw socket to the Wi-Fi driver.
```

Comment:
```suggestion
To receive the captured traffic in Monitor mode, the user application layer needs to open a raw socket to the Wi-Fi driver.
```

Reply:
Are there any specific raw socket configuration required to receive the capture traffic in mOnitor mode? If so, it would be nice to explain that here.

Reply:
It is a regular raw socket with ETH_P_ALL and SOCK_RAW input parameters for the socket fd API. Nothing different. But let me see how i can elaborate further